packages/site/src/layout/code-theme.css

```diff
@@ -12,6 +12,14 @@ pre {
   --color-code--teal: var(--color-base-teal--600);
 }
 
+:root[data-theme="dark"] pre {
```

Comment:
I think this might only apply to Storybook? It's in the docs directory but all of this was initially written 2+ weeks ago so this may not even be in use anymore 🤔